psalm-baseline.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove the baseline and put an inline `@psalm-suppress` to the line where this occurs.

Reason:
- baselined errors won't be highlighted by psalm
- if psalm options "reportUnusedPsalmSuppress" is enabled, updates of the laminas-router might make the suppression redundant and we can remove it while also optimizing the code

So I'd say enable that option and inline suppress the error, way more helpful than a baseline PLUS we do have a project without a baseline at all 👍🏼